### ISSUE TITLE
[Create trashpoint] User cannot add additional tags in other languages

### ISSUE THREAD

**Issue by darjalsk** [letsdoitworld/World-Cleanup-Day#279]
**Device:** iPhone 8+ 11.3.1
**Pre-conditions:** Release build, branch hotfix/blue_screen

**Steps to reproduce:**
Log in to the app as volunteer
Create a trashpoint -> add additional tag in other language.

**Actual result:** User CANNOT add additional tags in other languages.
**Expected result:** User can add additional tags in other languages.

**Comment by lynxlynxlynx** [comment]
works here, though the logic is a bit user-unfriendly. On android you have to let the keyboard close itself ("enter" button), just moving out of focus won't save them.

**Comment by darjalsk** [comment]
That’s right. User cannot add additional tag in any non-Latin languages (I tried Russian and Chinese) the way described above.

**Comment by KristiinaKerge** [comment]
is the same logic present in new version

**Comment by Dasha1990** [comment]
yep, it was moved to current app
